dev/DynamicDependency/API/MsixDynamicDependency.h

```diff
@@ -11,6 +11,18 @@
 /// MSIX Dynamic Dependency HRESULT: Windows App Runtime is not in the package graph.
 #define MDD_E_WINDOWSAPPRUNTIME_NOT_IN_PACKAGE_GRAPH    _HRESULT_TYPEDEF_(0x80040001L)
 
+/// MSIX Dynamic Dependency: Bootstrap initialization is scanning for an applicable DynamicDependencyLifetimeManager (DDLM) package
+#define MDD_E_BOOTSTRAP_INITALIZE_SCAN_FOR_DDLM         _HRESULT_TYPEDEF_(0x80040010L)
```

Comment:
Applies throughout

```suggestion
#define MDD_E_BOOTSTRAP_INITIALIZE_SCAN_FOR_DDLM         _HRESULT_TYPEDEF_(0x80040010L)
```

Reply:
Also search for "Intitialize" in the error outputs.